rationale_net/train/train.py

```diff
@@ -258,16 +264,23 @@ def run_epoch(data_loader, train_model, model, gen, optimizer, step, args):
         epoch_stat['k_selection_loss'] = np.mean(k_selection_losses)
         epoch_stat['k_continuity_loss'] = np.mean(k_continuity_losses)
 
-    return epoch_stat, step,  losses, preds, golds
+    return epoch_stat, step, losses, preds, golds
 
 
-def get_loss(logit,y, args ):
+def get_loss(logit,y, args):
     if args.objective == 'cross_entropy':
+        if args.use_as_classifier == False:
```

Comment:
maybe rename this to use as tagger. Use_as_classifier means something else in the larger rationale code base, because sometimes i use Encoders just to get a feature represetnation